compiler/luci-interpreter/src/kernels/Elu.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "kernels/Elu.h"
+#include "kernels/Utils.h"
+
+#include <tensorflow/lite/kernels/internal/optimized/optimized_ops.h>
+
+#include <stdexcept>
+
+namespace luci_interpreter
+{
+
+namespace kernels
+{
+
+Elu::Elu(const Tensor *input, Tensor *output) : _input(input), _output(output) {}
+
+void Elu::configure()
+{
+  assert(_input->element_type() == _output->element_type());
+  int dims = _input->shape().num_dims();
+  Shape output_shape(dims);
+  for (int i = 0; i < dims; i++)
+    output_shape.dim(i) = _input->shape().dim(i);
+  _output->resize(output_shape);
```

Comment:
Could be just
```suggestion
  _output->resize(_input->shape());
```

Reply:
this PR is merged. so I create new PR for your suggestion. plz check #2306